plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/server/LocalAppEngineServerLaunchConfigurationDelegate.java

```diff
@@ -94,7 +94,8 @@
 public class LocalAppEngineServerLaunchConfigurationDelegate
     extends AbstractJavaLaunchConfigurationDelegate {
 
-  private static final boolean DEV_APPSERVER2 = false;
+  static final boolean DEV_APPSERVER2 =
```

Comment:
Ping. I'm still not sold on the idea of condtionally enabling devappserver2 at this point int time. IMHO, we shoudl stick to devapopserver1 until devappserver2 is ready.